packages/cmd/src/profiles/CommandProfiles.ts

```diff
@@ -110,6 +176,17 @@ export class CommandProfiles {
         return this.mMap;
     }
 
+    /**
+     * Internal accessor for the meta map
+     * @readonly
+     * @private
+     * @type {Map<string, IProfileLoaded[]>} - The profile Map
+     * @memberof CommandProfiles
+     */
+    private get metaMap(): Map<string, IProfileLoaded[]> {
```

Comment:
Curious what is the purpose of the profile `metaMap`? It seems to map profiles based on their type, but I don't see it referenced anywhere in this branch or the `initial-token-support` branch of Zowe CLI.

Reply:
This allows for things like `this.metaMap.get(type)[0];` due to the capabilities of the `getter`. However, not sure if we really need one if it's going to be `private` to the class

Reply:
The `private` getter `metaMap` is used in the public `getMeta`.  `getMeta` is used in the future login handler (indirectly in the base handler).

I could remove this since it doesn't make sense.  I added it just to fit in the existing coding pattern of the file

Reply:
do you have a preference on keeping internal accessor / getters or removing them?

Reply:
I don't mind keeping it (to be consistent with the `coding pattern` of the file 😋

Reply:
ok perfect